homeworks/ramiro-miguez/01-html/contact.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Contact Me</title>
+  </head>
+  <body>
+    <nav>
+      <a href="index.html">Home</a> |
+      <a href="contact.html">Contact</a>
+    </nav>
+    <hr />
+    <section>
+      <h1>Contact Me</h1>
+      <form action="" method="post">
+        <div>
+          <label for="fname">Full name*</label>
```

Comment:
For next hws, try to use camelCase(google it) variant for your attribute/classes names... its more cleaner.. and also just and advice: never abbreviate words haha... in this case fullName its ok